### PR TITLE
upgrade/downgrade jobs reference kubernetes v1.8 version of etcd3

### LINKED ISSUE #9679: upgrade/downgrade jobs reference kubernetes v1.8 version of etcd3
```
      # TODO: This is pinned to the 1.8 version of etcd. 1.9 was changed to
      # 3.1.x. ETCD doesn't downgrade minor versions. So for downgrades, we
      # pin this to the lower version. The long term fix is to change
      # downgrade/upgrade to not upgrade/downgrade etcd.
      - --env=TEST_ETCD_VERSION=3.0.17
```

So this is a fun comment to find in the v1.13 release cycle
/milestone v1.13
/priority important-soon
/kind cleanup
/sig release
/sig cluster-lifecycle
/sig gcp

These jobs seem to be ownerless, and it's unclear to me what the solution here is, hence why I'm opening an issue.  Options:
- change TEST_ETCD_VERSION to the last 3.1.x release
- change TEST_ETCD_VERSION to whatever version was used by kubernetes v1.10
- change TEST_ETCD_VERSION to whatever version was used by kubernetes v1.11
- remove TEST_ETCD_VERSION entirely
- go see if the longterm fix is still valid, and/or how doable it is

### PR DESCRIPTION
**Pull request by RA489**
upgrade/downgrade jobs reference kubernetes v1.8 version of etcd3
fixes #9679

### CODE REVIEW

**Comment by timothysc** [comment]
/assign @roberthbailey  
/unassign @timothysc 

These are gce tests, which are owned by google.

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml [hunk 1]
```diff
@@ -300,7 +300,7 @@ periodics:
       # 3.1.x. ETCD doesn't downgrade minor versions. So for downgrades, we
       # pin this to the lower version. The long term fix is to change
       # downgrade/upgrade to not upgrade/downgrade etcd.
-      - --env=TEST_ETCD_VERSION=3.0.17
+      - --env=TEST_ETCD_VERSION=3.1.10
```
Comment (liggitt) [inline config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: do we want this to match the [currently recommended minimum version](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#prerequisites) (3.2.10)?

**Comment by AishSundar** [comment]
This seems to be a late landing change with potential to disrupt CI Signal. we never seem to have updated this for releases in the recent past. Do we have any evidence from logging to show the tests are still using the old etcd version?

/cc @jberkus

**Comment by AishSundar** [comment]
/hold
I am wondering if its worth holding this PR, until all other pending test fixes are in and we get a upgrade/downgrade signal. This could mean delaying this PR for a day, but we atleast wont risk the CI signal for the pending fixes and enable us to isolate any failures that result from this version change.

**Comment by spiffxp** [comment]
I appreciate that this is a first-time contribution that is more than a typo fix.  However, I would prefer that we not merge this so late during the 1.13 release cycle unless there is a demonstrated need to do so that is worth the potential disruption in CI signal.  I would also like to see some reasoning explaining why this option was chosen over the other options presented in #9679

**Comment by roberthbailey** [comment]
/assign @jpbetz 

(adding in an etcd expert)

/cc @krousey 

Also adding kris who might be able to provide some helpful historical context.

File: config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml (hunk 1 above)
Comment (jpbetz) [reply config/jobs/kubernetes/sig-cluster-lifecycle/k8s-upgrade-gce.yaml]: Yes, 3.2.10 please

**Comment by spiffxp** [comment]
pinged for update on https://github.com/kubernetes/test-infra/issues/9679, I'm still at the "this looks reasonable, but is this kicking the can down the road, or should we be doing more to address this?" point I was at during the 1.13 release cycle

**Comment by BenTheElder** [comment]
Seems like we should do it early in a cycle, it's still pretty early?
Not sure how long the road we're kicking it on is, but 3.0.17 looks to be 2 years old now.

**Comment by RA489** [comment]
@spiffxp any update on this?

**Comment by RA489** [comment]
we can close this issues if no updates.
cc @spiffxp

**Pull request closed without merging**